nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -218,6 +218,42 @@ def get_tarred_dataset(
 
         datasets.append(dataset)
 
+    if len(datasets) > 1:
+        if config.get('bucketing_batch_size', None) is not None and config['bucketing_batch_size'] > 0:
+            if config['batch_size'] != 1:
+                raise ValueError(
+                    f"batch_size should be set to one when bucketing_batch_size is set and adaptive bucketing is enabled (batch_size={config['batch_size']}!"
+                )
+
+            if type(config['bucketing_batch_size']) == int:
+                bucketing_batch_sizes = []
+                for idx in range(len(datasets)):
+                    scale_factor = len(datasets) - idx
+                    bucketing_batch_sizes.append(scale_factor * config['bucketing_batch_size'])
+            elif type(config['bucketing_batch_size']):
+                bucketing_batch_sizes = config['bucketing_batch_size']
+            else:
+                raise ValueError(
+                    f"bucketing_batch_size should be an integer or a list (bucketing_batch_size={config['bucketing_batch_size']})!"
+                )
+
+            if len(bucketing_batch_sizes) != len(datasets):
+                logging.warning(
+                    f"batch_size should have the same length as the number of buckets ({len(bucketing_batch_sizes)}!={len(datasets)}) "
+                )
+            logging.warning(
```

Comment:
Why is this a warning? It should be info

Reply:
fixed.